client/signup/steps/about/index.jsx

```diff
@@ -566,6 +582,8 @@ class AboutStep extends Component {
 				fallbackSubHeaderText={ subHeaderText }
 				signupProgress={ signupProgress }
 				stepContent={ this.renderContent() }
+				allowBackFirstStep={ allowBackFirstStep }
+				backUrl={ backUrl }
```

Comment:
It's a pity we can reliably pull the referring link from history, as we could then allow the back link to do what it says: go back, and not return to the dashboard. I tried with `page` and `document.referrer` and got mixed results.

Having said that, `<StepWrapper />` allows us to pass a custom back button label, so we might want to explicitly communicate to the user what's going to happen.

```
<StepWrapper
  ...
   backLabelText={ hasMultipleSites ? translate( 'Back to dashboard' ) : false }
/>
```

That might be a suitable compromise. Also,  _'Back to dashboard'_ has already been translated :)

Will we also add this feature to the Site Topic step? The AB test kicked in for me the first time I tested this branch, and it took me a few seconds to realize why I couldn't see the back link! :)

Reply:
> I tried with page and document.referrer and got mixed results.

Same. :-/ But I think changing the title to be clear is a solid compromise! I've updated the PR with that text.

> Will we also add this feature to the Site Topic step? The AB test kicked in for me the first time I tested this branch, and it took me a few seconds to realize why I couldn't see the back link! :)

That's a good idea! I can create a separate PR for that.